config/7.11.0/common.yaml

```diff
@@ -940,6 +940,24 @@ servers:
                       value: "[[.Kafka.MaxBlockMs]]"
                       #[[end]]
                     #[[end]]
+                    #[[if .KafkaJbpm]]
+                      #[[if .KafkaJbpm.BootstrapServers]]
+                    - name: KIE_SERVER_KAFKA_JBPM_BOOTSTRAP_SERVERS
+                      value: "[[.KafkaJbpm.BootstrapServers]]"
+                      #[[end]]
+                      #[[if .KafkaJbpm.Acks]]
+                    - name: KIE_SERVER_KAFKA_JBPM_ACKS
+                      value: "[[.KafkaJbpm.Acks]]"
+                      #[[end]]
+                      #[[if .KafkaJbpm.ClientID]]
+                    - name: KIE_SERVER_KAFKA_JBPM_CLIENT_ID
+                      value: "[[.KafkaJbpm.ClientID]]"
+                      #[[end]]
+                      #[[if .KafkaJbpm.MaxBlockMs]]
+                    - name: KIE_SERVER_KAFKA_JBPM_MAX_BLOCK_MS
+                      value: "[[.KafkaJbpm.MaxBlockMs]]"
```

Comment:
same as for the KieApp objects, I think that it would be good to use full names also for Kie Server env. variables. So instead of KAFKA_JBPM to use full naming -> KAFKA_JBPM_EVENT_EMITTER

Reply:
ok